src/js/base/Pagination.tsx

```diff
@@ -63,6 +65,7 @@ interface PaginationProps {
 }
 
 export const Pagination = ({
+    boxGroup,
```

Comment:
This isn't a good solution. Flags are almost always bad.

What about just putting a `margin-top` on `PaginationBox` instead and removing `PaginationContainer` entirely?